DashAI/front/src/components/DatasetsTable.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+
+import { DataGrid, GridActionsCellItem } from "@mui/x-data-grid";
+import DeleteIcon from "@mui/icons-material/Delete";
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+import {
+  Button,
+  Grid,
+  Paper,
+  Typography,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "@mui/material";
+
+function EditDataset() {
+  // const [open, setOpen] = React.useState(false);
+  const handleEdit = () => {};
+  return (
+    <GridActionsCellItem
+      key="edit-button"
+      icon={<EditIcon />}
+      label="Edit"
+      onClick={handleEdit}
+      sx={{ color: "#f1ae61" }}
+    />
+  );
+}
+
+function DeleteDataset({ deleteFromTable }) {
```

Comment:
Perhaps this component could be in another file. I think it looks very messy here.

Something like `DeleteDatasetModal`.

Reply:
I moved those components to new files, so now `DatasetsTable.jsx` contains only one component.

Reply:
Great!